testData/solutions/win32.json5

```diff
@@ -5807,6 +6472,458 @@
+                            "type": "fluid.transforms.value",
+                            "inputPath": "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.ScrollLines",
+                            "outputPath": "value"
+                        },
+                        "path": {
+                            "transform": {
+                                "type": "fluid.transforms.literalValue",
+                                "input": {
+                                    "get": "pvParam",
+                                    "set": "uiParam"
+                                }
+                            }
+                        }
+                    }
+                },
+                "inverseCapabilitiesTransformations": {
+                    "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.ScrollLines": "ScrollWheelModeConfig.value"
+                }
+            },
+            "configureScrollInactiveWindow": {
+                "type": "gpii.windows.spiSettingsHandler",
+                "liveness": "live",
+                "options": {
+                    "getAction": "SPI_GETMOUSEWHEELROUTING",
+                    "setAction": "SPI_SETMOUSEWHEELROUTING",
+                    "uiParam": 0,
+                    "pvParam": {
+                        "type": "UINT"
+                    },
+                    "fWinIni": "SPIF_UPDATEINIFILE|SPIF_SENDCHANGE",
+                    "verifySettings": true
+                },
+                "supportedSettings": {
+                    "ScrollFocusRoutingConfig": {
+                        "schema": {
+                            "title": "Mouse scroll focus routing",
+                            "description": "Changes the way mouse scrolling is passed to applications",
+                            "type": "string",
+                            "enum": ["RoutingFocus", "RoutingHybrid"]
+                        }
+                    }
+                },
+                "capabilitiesTransformations": {
+                    "ScrollFocusRoutingConfig": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "defaultInput": {
+                                "transform": {
+                                    "type": "fluid.transforms.value",
+                                    "inputPath": "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.ScrollRouting"
+                                }
+                            },
+                            "defaultOutputPath": "value",
+                            "match": {
+                                "RoutingFocus": 0,
+                                "RoutingHybrid": 1
+                            },
+                            "noMatch": 0
+                        },
+                        "path": {
+                            "transform": {
```

Comment:
This can just be literalValue: "pvParam"

Reply:
Changed all the literalValue to the simpler version.